tests/integration/main.go

```diff
@@ -24,46 +26,63 @@ var localSdkPath = flag.String("local-sdk-path", "",
 func main() {
 	flag.Parse()
 
-	// wg waits for all runners to complete
-	var wg sync.WaitGroup
-
-	start := time.Now()
-	tr := DefaultTestRun()
-	tr.SetLocalSDKPath(*localSdkPath)
-	tr.ValidateStringLiterals()
-	tr.startDocker()
-
-	dmc := DemocracyTestRun()
-	dmc.SetLocalSDKPath(*localSdkPath)
-	dmc.ValidateStringLiterals()
-	dmc.startDocker()
-
-	slash := SlashThrottleTestRun()
-	slash.SetLocalSDKPath(*localSdkPath)
-	slash.ValidateStringLiterals()
-	slash.startDocker()
-
-	if *multiconsumer {
-		mul := MultiConsumerTestRun()
-		mul.SetLocalSDKPath(*localSdkPath)
-		mul.ValidateStringLiterals()
-		mul.startDocker()
+	if happyPathOnly != nil && *happyPathOnly {
+		fmt.Println("=============== running happy path only ===============")
+		tr := DefaultTestRun()
+		tr.SetLocalSDKPath(*localSdkPath)
+		tr.ValidateStringLiterals()
```

Comment:
These four lines might be reformatted to:
```
tr.Run(happyPathSteps, *localSdkPath)
```

Reply:
Added